### PR TITLE
Add reanalyze button for projects

### PR DESCRIPTION
**Pull request by valentijnscholten**
This PR adds a button to the Project Audit Vulnerabilities tab to trigger a vulnerability analysis via `/api/v1/findings/project/{uuid}/analyze`. 

Accompanying https://github.com/DependencyTrack/dependency-track/pull/2128

Signed-off-by: Valentijn Scholten <valentijnscholten@gmail.com>

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -1,4 +1,4 @@
 NODE_ENV=development
 VUE_APP_I18N_LOCALE=en
 VUE_APP_I18N_FALLBACK_LOCALE=en
-VUE_APP_SERVER_URL=http://localhost:8080
+VUE_APP_SERVER_URL=http://172.22.38.80:8080
```
Comment (nscuro) [inline .env.development:4]: Whoops!

File: public/static/config.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
-  "API_BASE_URL": "",
+//  "API_BASE_URL": "https://dependencytrack-eindhoven.isaac.nl",
+  "API_BASE_URL": "http://172.25.253.209:8080",
```
Comment (nscuro) [inline public/static/config.json:3]: That shouldn't be here.

File: src/views/portfolio/projects/ProjectFindings.vue [hunk 3]
```diff
@@ -458,6 +466,13 @@
           link.click();
         });
       },
+      reAnalyze: function (data) {
+        let analyzeUrl = `${this.$api.BASE_URL}/${this.$api.URL_FINDING}/project/${this.uuid}/analyze`
+        this.axios.post(analyzeUrl).then((response) => {
+          //ignore token from response, don't wait for completion
```
Comment (nscuro) [inline src/views/portfolio/projects/ProjectFindings.vue:472]: Would be good to also display a toast so users know that something is actually happening.

Similar to how it's done for the metrics refresh button:

https://github.com/DependencyTrack/frontend/blob/9881a1d8655b261fedd4a5d53950d1351f1cb0da/src/views/Dashboard.vue#L212-L217

**Review by nscuro — CHANGES_REQUESTED** [review]

File: .env.development (hunk 1 above)
Comment (valentijnscholten) [reply .env.development]: `git add -u` all the things!

Comment (valentijnscholten) [reply .env.development]: fixed

File: public/static/config.json (hunk 2 above)
Comment (valentijnscholten) [reply public/static/config.json]: fixed

File: src/views/portfolio/projects/ProjectFindings.vue (hunk 3 above)
Comment (valentijnscholten) [reply src/views/portfolio/projects/ProjectFindings.vue:473]: Good suggestion, works now.

**Review by nscuro — APPROVED** [review]

**Pull request merged**